### PR TITLE
Hide PR action box for logged out users

### PR DESCRIPTION
**Pull request by wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf**
#### TL;DR
As consulted with @zeripath , I figured that casually browsing users don't need the info about PR mergeability, hence we don't show them the merge box thing.
#### Longer-ish story:
To name a particularly odd case, a message saying "`signing.wont_sign.%!s(<nil>)`" - as due to a missing signing key - was shown (pic 1), even though there in fact *does* exist a default signing key (pic 3) which is why the message kind of comes across as confusing.
It might therefore be safer (read: less confusing) to not display the mergeability status to a casual user (as in pic 2).

Before:
before

After:
after

There is a signing key:
thereisasigningkey

### CODE REVIEW

**Review by CirnoT — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Comment by zeripath** [comment]
@wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf I think #12581 might be a better solution - could you take a look before anyone merges.

**Review by techknowlogick — CHANGES_REQUESTED** [review]
Blocking so I don't accidentally merge

**Comment by wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf** [comment]
> @wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf I think #12581 might be a better solution - could you take a look before anyone merges.

I agree, closing #12564 (this) in favour of #12581 🚀
